src/gemdat/orientations.py

```diff
@@ -241,6 +240,40 @@ def transform(self, matrix: np.ndarray) -> Orientations:
 
         return replace(self, in_vectors=vectors)
 
+    def get_vectors_spherical(self, degrees: bool = True) -> np.ndarray:
+        """Return vectors in spherical coordinates.
+
+        Parameters
+        ----------
+        degrees : bool
+            If True, return angles as degrees, else radians
+
+        Returns
+        -------
+        np.array
+            azimuth, elevation, length
+        """
+        return cartesian_to_spherical(self.vectors)
```

Comment:
Maybe this can also return an `Orientation` doing something like
```suggestion
        return replace(self, in_vectors=cartesian_to_spherical(self.vectors))
```
similarly to `normalize`, `transform` and `symmetrized`.

Reply:
Maybe it makes more sense to have this as a property:

```Orientations.vectors_spherical```

Reply:
Ah but that loses the degrees flag 🤔

Reply:
The degrees flag was not passed, so that seems like a bug anyway.